src/HomeComponent/HomeComponent.js

```diff
@@ -4,62 +4,87 @@ import { Link } from 'react-router-dom';
 import { getObject } from '../exerciseStorageService';
 
 function HomeComponent () {
+  function weekStatus (n) {
+    const exerciseData = getObject();
+    const weekOneDayOneIsFinished = !!(exerciseData.finished.find(x => x[0] == n && x[1] == 1));
+    const weekOneDayTwoIsFinished = !!(exerciseData.finished.find(x => x[0] == n && x[1] == 2));
+    const weekOneDayThreeIsFinished = !!(exerciseData.finished.find(x => x[0] == n && x[1] == 3));
+
+    let weekOneStatus = 0;
+    if (weekOneDayOneIsFinished) {
```

Comment:
I think it is better if we start by checking if the last day is finished:

 if (weekOneDayThreeIsFinished) {
      weekOneStatus = 3;
      return weekOneStatus;
    }
   if (weekOneDayTwoIsFinished) {
      weekOneStatus = 2;
      return weekOneStatus;
    }
..

since if week 1 2 and 3 are finished, it will always return week one status as 1. Totally missed this.